internal/parse/parse.go

```diff
@@ -34,3 +36,47 @@ func Image(s string) error {
 
 	return nil
 }
+
+// DeploymentLogsForString parses the container logs of the specified ns/name
+// deployment, returning true if the string was found.
+func DeploymentLogsForString(ns, name, container, expectedString string) (bool, error) {
+	cmdPath, err := exec.LookPath("kubectl")
+	if err != nil {
+		return false, err
+	}
+	slashName := fmt.Sprintf("deployment/%s", name)
+	nsFlag := fmt.Sprintf("--namespace=%s", ns)
+	args := []string{"logs", slashName, "-c", container, nsFlag}
+	found, err := lookForString(cmdPath, args, expectedString)
+	if err != nil {
+		return false, err
+	}
+	if found {
+		return true, nil
+	}
+	return false, nil
+}
+
+// lookForString looks for the given string using cmd and args, returning
+// true if the string was found.
+func lookForString(cmd string, args []string, expectedString string) (bool, error) {
+	result, err := runCmd(cmd, args)
+	if err != nil {
+		return false, err
+	}
+	if strings.Contains(result, expectedString) {
+		return true, nil
+	}
+	return false, nil
+}
+
+// runCmd runs command cmd with arguments args and returns the output
+// of the command or an error.
+func runCmd(cmd string, args []string) (string, error) {
+	execCmd := exec.Command(cmd, args...)
```

Comment:
I looked through the `exec.Command` code and I don't see any mechanisms to prevent an arg that is prefixed by `/` or `.`. I've updated `runCmd()` based on your feedback.